fbw-common/src/wasm/systems/systems/src/hydraulic/trimmable_horizontal_stabilizer.rs

```diff
@@ -656,22 +729,22 @@ impl TrimmableHorizontalStabilizerHydraulics {
     pub fn update(
         &mut self,
         context: &UpdateContext,
+        controllers: &[impl ThsMotorController; 2],
         pressures: [Pressure; 2],
-        trim_actuator_output_position_normalized: Ratio,
     ) {
-        let deflection_demand = self.deflection_range
-            * trim_actuator_output_position_normalized.get::<ratio>()
-            + self.min_deflection;
-
-        self.update_spool_valve_lock_position(deflection_demand);
+        // Both controllers send the same position command for the A320, which is the aircraft that will use the
+        // spool valve lock position. The A380 will not use it, since it doesn't have a manual trim wheel.
+        // So, we just use the first controller position.
+        self.update_spool_valve_lock_position(controllers[0].requested_position());
 
         for (motor_index, motor) in self.hydraulic_motors.iter_mut().enumerate() {
```

Comment:
I know the code was already there before but nevertheless:
```suggestion
        for ((motor, controller), pressure) in self
            .hydraulic_motors
            .iter_mut()
            .zip(controllers)
            .zip(pressures)
        {
            motor.update(
                context,
                controller.motor_should_activate(),
                self.actual_deflection,
                controller.requested_position(),
                pressure,
            );
        }
```
Unfortunately I can't select multiple lines in this instance.